spec/spec_helper.rb

```diff
@@ -1,5 +1,15 @@
 require 'simplecov'
 require 'simplecov-console'
+require 'rspec'
+require 'sinatra'
+require 'rubocop'
+require 'capybara'
+require 'selenium-webdriver'
+require 'capybara/rspec'
```

Comment:
just realised now that I dont have these in my spec_helper lol so not sure if I was actually testing the whole thing correctly :DDD